<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RevelationClass</class>
 <widget class="QWidget" name="RevelationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revelation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1">
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelTitleTodo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Todo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="taskWidgetTodo" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelTitleProgramming">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Doing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="taskWidgetProgramming" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelTitleTesting">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Testing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="taskWidgetTesting" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelTitleDone">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="taskWidgetDone" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
